blog/docs/releases/announcing-knative-v1-7-release.md

```diff
@@ -0,0 +1,113 @@
+**Authors: [Matthias Wessendorf](https://github.com/matzew) (Red Hat), [Dave Protasowski](https://github.com/dprotaso) (VMware)**
```

Comment:
Add a front matter in the first line like it is in v1.5

Example from 1.5
---
title: "v1.5 release"
linkTitle: "v1.5 release"
Author: "[dprotaso](https://twitter.com/lintinmybelly)"
Author handle: https://github.com/dprotaso
date: 2022-06-06
description: "Knative 1.5 release announcement"
type: "blog"
---

after it also add the authors and date like you already have

Reply:
@csantanapr I basically copied from 1.6 announcement done by @evankanderson, which also does not have this.

Reply:
@csantanapr I think this was the frontmatter from Hugo days that we no longer use?

Reply:
I do think the author names etc should be under the heading though, for this and the 1.6 post, it looks weird that they're above the page title